DancingGoat/Controllers/SampleSiteFilter.cs

```diff
@@ -0,0 +1,12 @@
+using KenticoCloud.Delivery;
+
+namespace DancingGoat.Controllers
+{
+    public class SampleSiteFilter : IQueryParameter

```

Comment:
This is not exactly filter, just parameter, please rename to SampleSiteParameter